resources/charlie/chainspec.toml

```diff
@@ -30,25 +30,25 @@ accounts_path = '/etc/casper/accounts.csv'
 # make key blocks for the upcoming eras.  If the era is over by the time we start the nodes, they'll not be able to
 # produce blocks in it, and there won't be a new era build either.  That means when a completely new network is started,
 # the genesis era start time has to be adjusted to be active at the time.
-genesis_era_start_timestamp = 1600218437000
-# Era duration defined as a fixed number of milliseconds.  604800000 ms = 1 week.
-era_duration_millis = 604800000
+genesis_era_start_timestamp = 0
```

Comment:
That must be the same as `timestamp`. (Duplicated; will be removed soon.)

Reply:
Oh, good catch.  Messed this up syncing other close value.